### DISCUSSION TITLE
Repeated reference to the same footer

### DISCUSSION THREAD

**Discussion by sapphire008** [requarks/wiki#3629] [Error / Bug Report]
**Describe the bug**
Referencing the same footer, e.g. `[^1]` results the second footer to be incorrectly formatted. For example, the first `[^1]` will be rendered normally as `[1]`, but the second `[^1]` will be rendered as `[1:1]`. Then in the footer / reference section, another "return" symbol is appended to the end of the line.

**To Reproduce**
* In markdown editor, write the first sentence, reference with `[^1]`
* Write another sentence, then reference the same reference with `[^1]`
* Write the footer in the bottom, starting with `[^1]: reference detail`

**Expected behavior**
Expect to have the same rendering of `[1]` when referencing the same reference in the footer

**Screenshots**

**Host Info (please complete the following information):**
 - Hosted on the server, with CloudLinux
 - Wiki.js version: [e.g. 2.5.170]
 - Database engine: mysql 10

**Comment by NGPixel** [comment]
The [markdown-it-footnote](https://github.com/markdown-it/markdown-it-footnote/issues) is implemented as-is, so I suggest checking with the module maintainers first on how to do it.

**Comment by sapphire008** [comment]
Just looking at the github issues pages of markdown-it-footnote reveals that this is a problem that wasn't fixed since 2014. Maybe footnote is not the same as reference. Any suggestions on how to manage reference section in the markdown page properly?
